src/tasks/AppleAppBuilder/Xcode.cs

```diff
@@ -18,9 +18,11 @@ public Xcode(string target)
         switch (Target)
         {
             case TargetNames.iOS:
+            case TargetNames.iOSsim:
                 SysRoot = Utils.RunProcess("xcrun", "--sdk iphoneos --show-sdk-path");
```

Comment:
this should be `iphonesimulator` or `appletvsimulator` for simulators. interesting that it still worked apparently?

Reply:
hmm actually looks like this isn't even used?